src/jllvm/materialization/ByteCodeCompileLayer.cpp

```diff
@@ -972,6 +1000,41 @@ void codeGenBody(llvm::Function* function, const Code& code, const ClassFile& cl
                 break;
             }
             case OpCodes::Return: builder.CreateRetVoid(); break;
+            case OpCodes::InvokeVirtual:
```

Comment:
Could you move this case and the associated code so that all the cases remain alphabetical? You'll need to put braces around the whole code here for the C++ compiler to allow it.